specification/azurestackhci/resource-manager/Microsoft.AzureStackHCI/stable/2023-09-01/examples/PutLogicalNetwork.json

```diff
@@ -0,0 +1,47 @@
+{
+  "parameters": {
+    "subscriptionId": "fd3c3665-1729-4b7b-9a38-238e83b0f98b",
+    "resourceGroupName": "test-rg",
+    "logicalNetworkName": "test-lnet",
+    "api-version": "2023-09-01",
+    "logicalNetworks": {
+      "location": "West US2",
+      "extendedLocation": {
+        "type": "CustomLocation",
+        "name": "/subscriptions/fd3c3665-1729-4b7b-9a38-238e83b0f98b/resourceGroups/dogfoodarc/providers/Microsoft.ExtendedLocation/customLocations/dogfood-location"
+      }
+    }
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/subscriptions/fd3c3665-1729-4b7b-9a38-238e83b0f98b/resourceGroups/dogfoodarc/providers/Microsoft.AzureStackHCI/logicalNetworks/test-lnet",
+        "name": "test-lnet",
+        "type": "Microsoft.AzureStackHCI/logicalNetworks",
+        "location": "West US2",
+        "extendedLocation": {
+          "name": "/subscriptions/fd3c3665-1729-4b7b-9a38-238e83b0f98b/resourceGroups/dogfoodarc/providers/Microsoft.ExtendedLocation/customLocations/dogfood-location",
+          "type": "CustomLocation"
+        },
+        "properties": {
+          "provisioningState": "Accepted"
+        }
+      }
+    },
+    "201": {
+      "body": {
+        "id": "/subscriptions/fd3c3665-1729-4b7b-9a38-238e83b0f98b/resourceGroups/dogfoodarc/providers/Microsoft.AzureStackHCI/logicalNetworks/test-lnet",
+        "name": "test-lnet",
+        "type": "Microsoft.AzureStackHCI/logicalNetworks",
+        "location": "West US2",
+        "extendedLocation": {
+          "name": "/subscriptions/fd3c3665-1729-4b7b-9a38-238e83b0f98b/resourceGroups/dogfoodarc/providers/Microsoft.ExtendedLocation/customLocations/dogfood-location",
+          "type": "CustomLocation"
```

Comment:
>CustomLocation [](http://example.com/codeflow?start=19&length=14)

seeing this brings up questions for me like, is it meant to imply the same type as
'Microsoft.ExtendedLocation/customLocations'
?
is it redundant, if its already part of the full resource id?
should it be camelCased?
Or are you PascalCasing it because you think of it more like an enum?
Is the enum going to be defined by AzureStackHCI, or who?

Reply:
It is defined as an enum here: https://github.com/Azure/azure-rest-api-specs-pr/blob/980cb0ae2dce82d554fd0925eb16dacad675d92f/specification/azurestackhci/resource-manager/Microsoft.AzureStackHCI/stable/2023-09-01/common.json#L42